go.mod

```diff
@@ -1,11 +1,12 @@
 module github.com/CosmWasm/wasmd
 
-go 1.16
+go 1.17
 
 require (
 	github.com/CosmWasm/wasmvm v1.0.0-beta
-	github.com/cosmos/cosmos-sdk v0.42.10
+	github.com/cosmos/cosmos-sdk v0.44.2
```

Comment:
We are not even looking at PRs to bump past 0.42 until we cut a 1.0 of wasmd.
We are currently at 0.20, which is compatible with cosmwasm 1.0-beta, but missing a few last items to fine tune.
We are also waiting until the cosmos hub adopts some version of Cosmos SDK to add support for it in wasmd... this may be soon for 0.44.x????